src/components/sidebar/right-sidebar/datepicker/DateSlider.css

```diff
@@ -93,3 +93,21 @@
 .slider-right-value {
     right: -4px;
 }
+
+.ui.fluid.input>input {
+    width: 40% !important;
+}
+
+.ui.input>input {
+    padding: 5px;
+}
+
+.space-between {
```

Comment:
Pretty sure bootstrap already has a utility class for space-between. so might not need this

Reply:
seems wonky when i remove it, not sure if the utility class does what we need so ill just stick with what we have for now 

![image](https://user-images.githubusercontent.com/33983945/148312750-362020fa-fd5c-496f-a6a1-2fc85e3baaa8.png)